tests/ltp/patches/ltp_readv_readv02_fix.patch

```diff
@@ -0,0 +1,38 @@
++ Test will be enabled once git isue 297 is fixed.
++ Issue 297: [Tests] lkl_access_ok() should return -1 on invalid access
++ https://github.com/lsds/sgx-lkl/issues/297
+diff --git a/testcases/kernel/syscalls/readv/readv02.c b/testcases/kernel/syscalls/readv/readv02.c
+index aa40e2c7d..cbdf35eb3 100644
+--- a/testcases/kernel/syscalls/readv/readv02.c
++++ b/testcases/kernel/syscalls/readv/readv02.c
+@@ -126,7 +126,7 @@ int main(int ac, char **av)
+                                 "value");
+                }
+
+-//test2:
++/*test2: // TODO: Enable once git issue 297 is fixed
```

Comment:
Please use `#if 0` to disable entire chunks or you potentially trigger nested comment errors on a future update.